### DISCUSSION TITLE
Oauth2 Implementation with VTEX

### DISCUSSION THREAD

**Discussion by SrinivasPeddi** [community/community#52744] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

Hi Team,

Present i am working on GIT hub login  integration with vtex through OAUTH2.
As per below git document i did configuration in VTEX side and GIT side with Oauth app creation.

https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps#device-flow

I am facing issue while redirecting url from git and receiving status as  "CanceledByUser".
PFA doc and .har files for error reference. 

Please share any other way or solution for this.
[GIT_OAUTH2_Config_Steps.docx](https://github.com/community/community/files/11224402/GIT_OAUTH2_Config_Steps.docx)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @SrinivasPeddi ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
